ffupdater/build.gradle

```diff
@@ -45,37 +45,38 @@ dependencies {
     implementation "com.github.topjohnwu.libsu:core:5.2.0"
 
     // for Shizuku installer
-    compileOnly "dev.rikka.hidden:stub:4.2.0"
-    implementation "dev.rikka.tools.refine:runtime:4.3.0"
-    def shizuku_version = "13.1.2"
+    compileOnly 'dev.rikka.hidden:stub:4.3.3'
+    implementation 'dev.rikka.tools.refine:runtime:4.4.0'
+    def shizuku_version = '13.1.5'
     implementation "dev.rikka.shizuku:api:${shizuku_version}"
     implementation "dev.rikka.shizuku:provider:${shizuku_version}"
     implementation "org.lsposed.hiddenapibypass:hiddenapibypass:4.3"
 
     // Aggregator dependency on JUnit api, engine, and params
-    testImplementation "org.junit.jupiter:junit-jupiter:5.9.3"
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.11.4'
     testImplementation "org.hamcrest:hamcrest-all:1.3"
-    testImplementation "io.mockk:mockk:1.13.5"
+    testImplementation 'io.mockk:mockk:1.13.14'
     testImplementation "io.github.ivanshafran:shared-preferences-mock:1.2.4"
-    testImplementation "org.slf4j:slf4j-simple:2.0.9"
+    testImplementation 'org.slf4j:slf4j-simple:2.0.16'
 }
 
 android {
     namespace = "de.marmaro.krt.ffupdater"
 
-    compileSdk 34
+    compileSdk 35
 
     defaultConfig {
         versionCode 177
         //  (incompatible changes) . (backwards compatible functionality) . (backwards compatible bug fixes)
-        versionName "79.3.1"
-        minSdk 21
-        targetSdk 34
+        versionName "79.3.2a0"
+        minSdk 28
+        targetSdk 35
```

Comment:
FYI changing the targetSdk is risky and should be done with care. I will check if there is some breaking changes in Android 15 https://developer.android.com/about/versions/15/behavior-changes-15